tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -1340,9 +1345,12 @@ <h2>Create a distance feature</h2>
                           <div class="feature-breakdown">
                             <div class="slider-label">Threshold for positive classification</div>
                             <div class="flex">
-                              <paper-slider class="slider" editable=true min="0" max="1" step="0.01"
-                                            immediate-value="{{overallThreshold}}" value="[[overallThreshold]]">
-                              </paper-slider>
+                              <template is="dom-repeat" items="{{overallThresholds}}" as="overallThreshold">
+                                <paper-slider class="slider" editable=true min="0" max="1" step="0.01"
```

Comment:
add label for sliders when more than one model?

Reply:
added.